README.md

```diff
@@ -1,7 +1,9 @@
 # Leaderboard-Setup-Project
- This is a Leaderboard API app that stores users score
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Additional description about the project and its features.
+This is a Leaderboard API app that stores users score
+![](leaderboardappsnapshot.PNG)
 
```

Comment:
- [ ] [OPTIONAL] It would be much nicer if you add Author section into your README.md. Other than that your project looks great! 👏